js/main.js

```diff
@@ -59,6 +89,30 @@ function renderEntry(entry) {
 }
 
 const $ul = document.querySelector('ul');
+$ul.addEventListener('click', clickPencil);
+
+const $title = document.querySelector('#title');
+const $notes = document.querySelector('#notes');
+const $h1 = document.querySelector('h1');
+
+function clickPencil(event) {
+  if (event.target.getAttribute('class') === 'fa fa-pencil fa-lg') {
+    viewSwap('entry-form');
+    for (let i = 0; i < data.entries.length; i++) {
+      if (
+        JSON.stringify(data.entries[i].entryId) ===
+        event.target.closest('li').getAttribute('data-entry-id')
```

Comment:
You shouldn't be using `JSON.stringify` here. We don't want to convert this to a JSON string. You just need to convert the string to a number. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/Number